### PR TITLE
no merge

### PR DESCRIPTION
**Pull request by kuzetsa**
Package-Manager: Portage-2.3.3, Repoman-2.3.1

### CODE REVIEW

File: sys-kernel/ck-sources/ck-sources-4.10.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+src_prepare() {
+
+#-- Comment out CK's EXTRAVERSION in Makefile ---------------------------------
+
+	# linux-info eclass cannot handle recursively expanded variables in Makefile #490328
+	sed -i -e 's/\(^EXTRAVERSION :=.*$\)/# \1/' "${S}/Makefile" || die
+
+	# EAPI 6: default action doesn't happen by default #612782
+	default
```
Comment (gktrk) [inline sys-kernel/ck-sources/ck-sources-4.10.3.ebuild]: We should call `kernel-2_src_prepare' here instead

File: sys-kernel/ck-sources/ck-sources-4.9.15.ebuild [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	use deblob && python-any-r1_pkg_setup
+	kernel-2_pkg_setup
+}
+
+src_prepare() {
+
+#-- Comment out CK's EXTRAVERSION in Makefile ---------------------------------
+
+	# linux-info eclass cannot handle recursively expanded variables in Makefile #490328
+	sed -i -e 's/\(^EXTRAVERSION :=.*$\)/# \1/' "${S}/Makefile" || die
```
Comment (gktrk) [inline sys-kernel/ck-sources/ck-sources-4.9.15.ebuild]: We should add a call to `kernel-2_src_prepare' here

**Comment by gktrk** [comment]
See: https://github.com/gentoo/gentoo/compare/75af8f6...3530806

**Comment by kuzetsa** [comment]
This is no longer a simple bump, so the commit message no longer makes sense. Please don't merge this until new commits (yours or mine - either way) are written.

**Pull request closed without merging**